x/rollapp/types/token_metadata.go

```diff
@@ -18,6 +18,7 @@ func (m *TokenMetadata) Validate() error {
 }
 
 // ConvertToBankMetadata converts TokenMetadata to Metadata of x/bank/types
+// TODO: there is no good reason we have a duplicate type, so we should just use the bank one always
 func (m *TokenMetadata) ConvertToBankMetadata() banktypes.Metadata {
```

Comment:
`TokenMetadata` and all related code can be removed

Reply:
great spot

I thought I had done that, maybe it got brought back when I messing around with git


Anyways, deleted thanks